src/ProjectTemplates/test/ItemTemplateTests/RazorComponentsTests.cs

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Templates.Test.Helpers;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Templates.Items.Test
+{
+    public class RazorComponentsTest
+    {
+        public RazorComponentsTest(ProjectFactoryFixture projectFactory, ITestOutputHelper output)
+        {
+            ProjectFactory = projectFactory;
+            Output = output;
+        }
+
+        public Project Project { get; set; }
+
+        public ProjectFactoryFixture ProjectFactory { get; }
+        public ITestOutputHelper Output { get; }
+
+        [Fact]
+        public async Task RazorComponentsItemTemplate()
+        {
+            Project = await ProjectFactory.GetOrCreateProject("razorcomponentitem", Output);
+
+            var createResult = await Project.RunDotNetNewAsync("razorcomponent --name Steve");
+            Assert.True(0 == createResult.ExitCode, ErrorMessages.GetFailedProcessMessage("create", Project, createResult));
+
+            Project.AssertFileExists("Steve.razor", shouldExist: true);
+            Assert.Contains("<h3>Steve</h3>", Project.ReadFile("Steve.razor"));
```

Comment:
I don't think these two asserts add any value. This feels like testing that a property has property behavior.

Reply:
`AssertFileExists` confirms that the output name gets changed and Assert.Contains confirms that the replacement of the `Index` value happens.